### ISSUE TITLE
Pytorch next steps

### ISSUE THREAD

**Issue by hhbyyh** [intel/analytics-zoo#1027]
Use this issue as a reminder to track the Pytorch-related work items

1. Support Table/multiple input and output models. [done]

2. JNI memory optimization. Avoid mem leak and unnecessary copy.

3. Scalability test for training and inference

3. Support export to Pytorch model.

5. Utility to convert data loader to RDD/DataFrame

6. RNN/LSTM training.

7. Confirm tracing batch normalization works for training.
